Source/Module/modules/audio/commands/PlayAudioFileCommand.cpp

```diff
@@ -102,7 +102,7 @@ void PlayAudioFileCommand::setAudioFile(File f)
 	if (reader != nullptr)
 	{
 		std::unique_ptr<AudioFormatReaderSource> newSource(new AudioFormatReaderSource(reader, true));
-		transportSource.setSource(newSource.get(), 0, nullptr, reader->sampleRate); 
+		transportSource.setSource(newSource.get(), 0, nullptr, reader->sampleRate, reader->numChannels);
```

Comment:
The optional argument for the channel count was not set, and so defaulted to 2 channels